### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by gerriejoubert**
Added a line for TCP 5061 on the customer firewall for inbound and outbound, for line 277

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -273,7 +273,8 @@ You must use the following ports for Microsoft 365 or Office 365 environments wh
 |Traffic|From|To|Source port|Destination port|
 |:--- |:--- |:--- |:--- |:--- |
 |SIP/TLS|SIP Proxy|SBC|1024 – 65535|Defined on the SBC (For Office 365 GCC High/DoD only port 5061 must be used)|
-SIP/TLS|SBC|SIP Proxy|Defined on the SBC|5061|
+SIP/TLS|SBC|SIP Proxy|Defined on the SBC|5061| 
+SIP/TLS|SBC|SIP Proxy|Defined on the customer Firewall|5061| (Ensure that TCP 5061 is open you the firewall for inbound and outbound traffic)|
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md:277]:
```suggestion
SIP/TLS|SBC|SIP Proxy|Defined on the customer Firewall|5061 (Ensure that TCP 5061 is open on the firewall for inbound and outbound traffic)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gerriejoubert, kindly commit the suggested change. Many thanks.

**Comment by gerriejoubert** [comment]
Morning

Your message is not very clear as to what is needed, please clarify.

Kind Regards,

Gerrie Joubert / Professional Services Associate
THINKTEL a division of DISTRIBUTEL
#1506, 101 street NW, Edmonton, AB, T5J-3P4

T +1 (780) 809-8002  M +1 (780) 777-1592
www.distributel.ca<http://www.distributel.ca/>

**Comment by JohanFreelancer9** [comment]
@gerriejoubert - You should see a "Commit" or "Commit changes" button next to or below the suggested changes. Please click on it to commit the changes. Thanks.

**Comment by gerriejoubert** [comment]
I did can you see it now?

Kind Regards,

Gerrie Joubert / Professional Services Associate
THINKTEL a division of DISTRIBUTEL
#1506, 101 street NW, Edmonton, AB, T5J-3P4

T +1 (780) 809-8002  M +1 (780) 777-1592
www.distributel.ca<http://www.distributel.ca/>

**Comment by JohanFreelancer9** [comment]
No, it is still not committed. Could you please try again?

**Comment by JohanFreelancer9** [comment]
@gerriejoubert - Thank you for addressing these changes in PR #7684. Looks good. Could you please just close this PR? Many thanks.

**Pull request closed without merging**